Demos/Raven.NET.Demo.Console/Program.cs

```diff
@@ -21,12 +21,13 @@ static void Main(string[] args)
                 .ConfigureServices((_, services) =>
                 {
                     services.ConfigureRaven(builder.Build());
```

Comment:
Maybe all of those registration can be part of Configure method? Now got this idea

Reply:
good idea. We could have just one extension method "AddRaven()" that will do all this work